src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/TransmissionStorage.cs

```diff
@@ -169,9 +178,18 @@ private static string GetUniqueFileName(string extension)
 
         private static Transmission LoadFromTransmissionFile(IPlatformFile file, out long fileSize)
         {
-            ChangeFileExtension(file, TemporaryFileExtension);
-            Transmission transmission = LoadFromTemporaryFile(file, out fileSize);
-            file.Delete();
+            Transmission transmission = null;
+            if (file.Exists)
+            {
+                ChangeFileExtension(file, TemporaryFileExtension);
+                transmission = LoadFromTemporaryFile(file, out fileSize);
+                file.Delete();
+            }
+            else
+            {
+                fileSize = 0;
```

Comment:
My understanding that before it would throw if file cannot be read. Will consumer of this function expect `null` as a result?